docs/guides/bulk/upload.md

```diff
@@ -0,0 +1,26 @@
+---
+heading: Bulk
+seo: Bulk | Bulk Download | How to Use Bulk Upload
+title: Upload
+description: How to use Bulk Upload
+layout: sidebarleft
+apis: API Docs
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 3
+---
+
+# Bulk Upload
+
+Unlike Bulk Downloads, using the bulk upload requires only a single call. The `POST /bulk/{objectName}` uploads a file which will then be processed by the bulk job.
+In most cases, a transformation will be required to map the field names in the file to the field names in the endpoint. The Bulk Upload is composed of the following parts:
```

Comment:
"In most cases, a transformation will be required to map the field names in the file to the field names in the endpoint."

This seems like a statement that might need some more information around it. Why? How?